src/order-and-subscriptions/_style.scss

```diff
@@ -0,0 +1,23 @@
+.page__order-and-subscriptions {
+  display: flex;
+  flex-direction: column;
+  gap: 2.5rem;
+
+  section, .section {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+
+    .section-gap-sm {
+      gap: 0.75rem;
+    }
+
+    h1, h2 {
+      margin-bottom: 0;
```

Comment:
please prefer to use `bootstrap` / `paragon` utility classes wherever possible. Here `margin-bottom` can be `@extend .mb-0`.

https://paragon-openedx.netlify.app/foundations/spacing
https://hackerthemes.com/bootstrap-cheatsheet/